### PR TITLE
OSDOCS-4564: renaming troubleshooting book and adding known issue

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
[OSDOCS-4564](https://issues.redhat.com//browse/OSDOCS-4564): renaming troubleshooting book and adding known issue of deleted iptables upon firewall restart

* MicroShift project

Version(s):
  * PR applies to: 4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4564

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@jogeo PTAL and suggest additions or subtractions to procedure as needed. Thank you!

**Comment by jogeo** [comment]
@ShaunaDiaz Step `2. Delete the ovn-master pod` and `3. Restart the DaemonSet pod` can be combined into one. A daemonSet describes the desired state of the openshift-ovn-kubernetes namespace, with respect to how many pods should be running. So, when the command described in step 2 is executed to delete the pod, the DaemonSet config automatically restarts it, to maintain the desired state. Therefore, step 3 is not required.

Step `4. Confirm that the iptables have reconciled by running the following command` would not be an 'oc' command. Rather the iptables defined on the host, where MicroShift is running, need to be inspected. 
`$ sudo iptables-save | grep NODEPORT
:OVN-KUBE-NODEPORT - [0:0]
-A PREROUTING -j OVN-KUBE-NODEPORT
-A OUTPUT -j OVN-KUBE-NODEPORT
-A OVN-KUBE-NODEPORT -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 30768 -j DNAT --to-destination 10.43.17.173:443                                                                                           
-A OVN-KUBE-NODEPORT -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 32122 -j DNAT --to-destination 10.43.17.173:80`

The only other nit I'll mention is that in order to run the 'oc' command as the examples show the KUBECONFIG environment variable must be set, prior to running 'oc' (e.g. export KUBECONFIG=$HOME/kubeconfig).

File: modules/microshift-ki-cni-iptables-deleted.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+// * microshift_support/microshift-version.adoc
+
+[id="microshift-ki-cni-iptables-deleted_{context}"]
+= Reloading the firewall deletes iptable rules
+
+OVN-Kubernetes handles incoming NodePort traffic by using iptable rules. When you reload firewall rules with the `firewall-cmd --reload` command, the iptable rules are deleted. This stops the NodePort service traffic and any other host traffic that uses iptable rules.
+//Q: Is firewall reload something that happens automatically and continuously? When does it happen?
+[id="microshift-ki-cni-iptables-deleted-workaround_{context}"]
+== Restarting the DaemonSet pod as a workaround
+To troubleshoot this issue, restart the ovnkube DaemonSet pod. Restarting the pod will trigger a reconciliation of the iptable rules.
```
Comment (mburke5678) [inline modules/microshift-ki-cni-iptables-deleted.adoc:12]: Is the pod named _ovnkube_? 
If not, s/ovnkube DaemonSet pod/ ovnkube daemon set pod
If so, s/ovnkube DaemonSet pod/`ovnkube` DaemonSet pod

File: modules/microshift-ki-cni-iptables-deleted.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+.Procedure
+
+. Stop the ovn-master pod:
++
+[source, terminal]
+----
+$ pod=$(oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master | awk -F " " '{print $1}')
+----
+
+. Delete the ovn-master pod:
```
Comment (mburke5678) [inline modules/microshift-ki-cni-iptables-deleted.adoc:30]: Here, also.

File: modules/microshift-ki-cni-iptables-deleted.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes CNI cluster network provider.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+. Stop the ovn-master pod:
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/microshift-ki-cni-iptables-deleted.adoc:23]: Add _Use the following command to_

File: modules/microshift-ki-cni-iptables-deleted.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/microshift-ki-cni-iptables-deleted.adoc:30]: Add _Use the following command to_

File: modules/microshift-ki-cni-iptables-deleted.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+----
+
+. Delete the ovn-master pod:
++
+[source, terminal]
+----
+$ oc -n openshift-ovn-kubernetes delete pod $pod
+----
++
+This step will automatically restart the DeamonSet pod, causing a reconciliation of the iptable rules.
```
Comment (mburke5678) [inline modules/microshift-ki-cni-iptables-deleted.adoc:37]: s/step/command
s/will automatically restart/automatically restarts

**Comment by mburke5678** [comment]
@ShaunaDiaz A few nits. Otherwise LGTM!

File: modules/microshift-ki-cni-iptables-deleted.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply modules/microshift-ki-cni-iptables-deleted.adoc:12]: No, each pod will have a suffix that isn't predictable.

File: modules/microshift-ki-cni-iptables-deleted.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-ki-cni-iptables-deleted.adoc:23]: ovn-master is the application

**Pull request merged**